### DISCUSSION TITLE
Warning when doing full home folder restore

### DISCUSSION THREAD

**Discussion by patogit** [borgbase/vorta#837] [General]
I haven't tried this because I don't want to mess up my system.

What happens if I extract an old settings.db that overwrites current settings.db? Say I have daily backups of `/home` from May 1 - 10, and today is May 11. Now I extract the May 5 archive, and it overwrites `settings.db` with the May 5 version, and thus the May 6-10 archives disappear from the list even though they're still there (see https://github.com/borgbase/vorta/issues/462#issuecomment-632733200). Plus Vorta just overwrote its own settings while running, which seems like a dangerous thing.

Is there some kind of protection against this? If not, I just have to make sure I always exclude `settings.db` when extracting a `/home` archive.

**Comment by m3nu** [comment]
Since the DB file is kept open by Vorta, it would probably keep using the old settings file, until you restart it.

You can always read the list of archives from Borg itself using the *Refresh* button.

That said, there are many many other apps you can break by overwriting them. So restoring a full home folder or essential setting should always be done from another account.

We could probably add a note about this in the docs.
